cypress/e2e/topics/copyTopicInCourseDetailPage.feature

```diff
@@ -24,7 +24,8 @@ Feature: Topics - Teacher copies a topic in the course
         When I select the three dot menu action 'copy' at topic index '0' in course detail page
         Then I see the progress bar
         Then I see the dialog for copying topic alert
-        Then I see the warning in the dialog that GeoGebra and Etherpad content were not copied
+        Then I see the warning in the dialog box
```

Comment:
IMO it would be better if we say 

- `I see the warning section in the dialog box` 
or
- `I see the warning section displayed in the dialog box`

 which makes things descriptive rather imperative